src/styles/main.scss

```diff
@@ -1 +1,26 @@
-// put code here
+@import "blocks/hand";
+@import "blocks/stopwatch";
+
+body {
+  min-height: 100vh;
+  margin: 0;
+  padding: 0;
+}
+
+@keyframes min {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes sec {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
no need to make two same keyframes.
you can use one keyframes for both arrows

Reply:
Consider resolving this comment